peaseq-case.wdl

```diff
@@ -513,89 +637,94 @@ workflow peaseq {
         #   FastQC on Merge BAM (AllMapped_<number>_mapped)
 
         # merge bam files and perform fasTQC if more than one is provided
-        call util.mergehtml as PE_mergehtml {
-            input:
-                htmlfiles=indv_PE_summarystats.xhtml,
-                txtfiles=indv_PE_summarystats.textfile,
-                default_location='SAMPLE',
-                outputfile = 'AllMapped_PEmode_peaseq-summary-stats.html'
+        call util.mergehtml as PE_mergehtml { input:
+            htmlfiles = indv_PE_summarystats.xhtml,
+            txtfiles = indv_PE_summarystats.textfile,
+            default_location = "SAMPLE",
+            outputfile = "AllMapped_PEmode_peaseq-summary-stats.html",
         }
 
-        call peaseq_util.pe_mergehtml as final_mergehtml {
-            input:
-                pe_htmlfiles=indv_PE_summarystats.xhtml,
-                pe_txtfiles=indv_PE_summarystats.textfile,
-                se_htmlfiles=indv_summarystats.xhtml,
-                se_txtfiles=indv_summarystats.textfile,
-                default_location='SAMPLE',
-                outputfile = 'AllSamples-summary-stats.html'
+        call peaseq_util.pe_mergehtml as final_mergehtml { input:
+            pe_htmlfiles = indv_PE_summarystats.xhtml,
+            pe_txtfiles = indv_PE_summarystats.textfile,
+            se_htmlfiles = indv_summarystats.xhtml,
+            se_txtfiles = indv_summarystats.textfile,
+            default_location = "SAMPLE",
+            outputfile = "AllSamples-summary-stats.html",
 
```

Comment:
hm. This blank should be removed, but I'm not quite sure if `format` should have removed it automatically 🤔 this is a corner case that could be worth revisiting